firefox_puppeteer/api/places.py

```diff
@@ -77,30 +77,29 @@ def is_bookmark_star_button_ready(self):
 
     def restore_default_bookmarks(self):
         """Restores the default bookmarks for the current profile."""
-        try:
-            return self.marionette.execute_async_script("""
-              Cu.import("resource://gre/modules/BookmarkHTMLUtils.jsm");
-              Cu.import("resource://gre/modules/Services.jsm");
-
-              // Default bookmarks.html file is stored inside omni.jar,
-              // so get it via a resource URI
-              let defaultBookmarks = 'resource:///defaults/profile/bookmarks.html';
-
-              let observer = {
+        retVal = self.marionette.execute_async_script("""
+            Cu.import("resource://gre/modules/BookmarkHTMLUtils.jsm");
+            Cu.import("resource://gre/modules/Services.jsm");
+            // Default bookmarks.html file is stored inside omni.jar,
+            // so get it via a resource URI
+            let defaultBookmarks = 'resource:///defaults/profile/bookmarks.html';
+            let observer = {
                 observe: function (aSubject, aTopic, aData) {
-                  Services.obs.removeObserver(observer, "bookmarks-restore-success");
+                Services.obs.removeObserver(observer, "bookmarks-restore-failed");
```

Comment:
Make sure to use the right indentation of 2 chars for Javascript code. Looks like because of the removal of try/except all the lines got a bit messed-up.